gvr-blurfilter/app/src/main/res/raw/gaussianblurhorz.fsh

```diff
@@ -0,0 +1,28 @@
+#extension GL_OES_EGL_image_external_essl3 : require

```

Comment:
I guess enabling both may be an option?
```
#extension GL_OES_EGL_image_external_essl3 : enable
#extension GL_OES_EGL_image_external : enable
```

Although after I did that I got ``0:21: S0001: No matching overload for function 'texture' found``.